core/src/main/java/org/sonarsource/sonarlint/core/websocket/SonarCloudWebSocket.java

```diff
@@ -38,8 +38,10 @@
 import org.sonarsource.sonarlint.core.websocket.parsing.QualityGateChangedEventParser;
 
 public class SonarCloudWebSocket {
-  public static final String WEBSOCKET_DEV_URL = "wss://squad-5-events-api.sc-dev.io/";
-  public static final String WEBSOCKET_URL = "wss://events-api.sonarcloud.io/";
+
+  public static String getUrl() {
+    return System.getProperty("sonarlint.internal.sonarcloud.websocket.url", "wss://events-api.sonarcloud.io/");
```

Comment:
cool! from where do we usually set this kind of properties?

Reply:
In your case you can insert `-Dsonarlint.internal.sonarcloud.websocket.url=blah` in the `java` command starting the LS